server/lib/scanners/jellyfin/index.ts

```diff
@@ -96,7 +96,7 @@ class JellyfinScanner {
           (MediaStream) => MediaStream.Type === 'Video'
         ).some((MediaStream) => {
           return (MediaStream.Width ?? 0) <= 2000;
-        });
+        }) || (MediaSource.Container && MediaSource.Container === "strm");
```

Comment:
The first condition here is not necessary.

```suggestion
        }) || MediaSource.Container === 'strm';
```